vela-templates/definitions/internal/trait/expose.cue

```diff
@@ -1,3 +1,7 @@
+import (
+	"strconv"
+)
+
 expose: {
```

Comment:
If this trait is ready, please remove the `ui-hidden` label.

Reply:
Why the previous version is not ready?

Reply:
No developer to test it.